admin_manual/installation/system_requirements.rst

```diff
@@ -13,13 +13,22 @@ Server
 ================= =============================================================
 Platform          Options
 ================= =============================================================
-Operating System  Ubuntu 16.04, Debian 7 and 8, SUSE Linux Enterprise Server 12 
-                  and 12 SP1, Red Hat Enterprise Linux/Centos 6.5 and 7 
+Operating System  Ubuntu 16.04 and 17.10; Debian 7, 8 and 9; SUSE Linux Enterprise Server 12 
+                  with SP2 and SP3; Red Hat Enterprise Linux/Centos 6.9, 7.3 and 7.4;
+                  Fedora 26 and 27; open Suse Tumbleween and Leap
 Database          MySQL or MariaDB 5.5+, Oracle 11g, PostgreSQL, & SQLite
 Web server        Apache 2.4 with ``prefork`` :ref:`apache-mpm-label` and ``mod_php``
 PHP Runtime       PHP (5.6+, 7.0, & 7.1)
 ================= =============================================================
 
+
+.. info:: 
+
+    For the future release of ownCloud 10.1, a minimum php version of 7.1 is needed.
+    If you use Ubuntu 16.04:
+    - php 7.1 is only available via ppa
+    - php 7.2 standard installable, but you have to install some mandatory modules yourself, like intl.
```

Comment:
pls take care: 
**not** all modules like ``intl`` mandatory for owncloud are availabe for this combination.

Reply:
can you add some commas? :D

I wrote : In ubuntu 16.04, php 7.2 is available but you have to install some modules, that are mandatory for owncloud to work, manually, like intl.

You wrote: Not all modules, like intl, that are mandatory for owncloud, are available with Ubuntu 16.04 and php 7.2.

Did I miss something?

Reply:
What you have written is clear enough. I'd leave it as is.

Reply:
I have to make some changes, don't merge :)

Reply:
There is a slightly difference in
``you have to install some modules``
versus
``Not all modules are available``
Having Ubuntu 16.04, you can install php 7.2 but you **cant** run owncloud because of modules not available in this combination...

pls see: https://packages.ubuntu.com/search?keywords=php-intl
php7.2-intl is available ONLY for bionic (18.04, LTS, estimated release date 28. April 2018)...

Reply:
So where do you get this specific module then from? 

Or is it simply not available and ergo you should not install php 7.2 on ubuntu 16.04?

Reply:
simply not available.
16.04: 
php 7.0 fully ok as long you are under oC 10.1
php 7.1 via ppa, works with 10.1 (I tested this!)
php 7.2 installable but not working with oC 10.1 because of not available but mandatory modules

You can see the php thing in the package link above...

Reply:
So, we could just write - don't install 7.2.

I just trying to understand you, and making it easier for the users who will read this :)

Reply:
Or 10.1 doesn't work with php 7.2

Reply:
I am not sure about this one. I thought it is... needs double checking

Reply:
Well, if it does not have a module that is required for ownCloud to work, why can't we say that users should not install 7.2 with 10.1?

Reply:
Because this is limited to the combination of Ubuntu 16.04 - pls see my comment above.

Reply:
Suggestion:
Please consider that not all mandatory php modules necessary for ownCloud are available for php 7.2 running on Ubuntu 16.04.